Makefile

```diff
@@ -1,36 +1,44 @@
-.PHONY: setup rebuild start superuser migrations migrate pull-production-data pull-production-media pull-staging-data pull-staging-media
-
-setup:
-	make rebuild
+.PHONY: setup
+setup: rebuild
 	docker-compose run web django-admin migrate
 	docker-compose run web django-admin createcachetable
 
+.PHONY: rebuild
 rebuild:
-	bash -c "docker-compose build --build-arg UID=$$(id -u) --build-arg DOCKER_GID=$$(ls -ln /var/run/docker.sock | awk '{print $$4}')"
+	bash -c "docker-compose build --build-arg UID=$$(id -u) --build-arg GID=$$(id -g)"
```

Comment:
```
 => ERROR [backend 5/6] RUN groupadd --gid 20 wagtailio     && useradd --create-home --uid 503 --gid 20 wagtailio                                                   0.2s
------
 > [backend 5/6] RUN groupadd --gid 20 wagtailio     && useradd --create-home --uid 503 --gid 20 wagtailio:
#0 0.201 groupadd: GID '20' already exists
------
failed to solve: executor failed running [/bin/sh -c groupadd --gid $GID wagtailio     && useradd --create-home --uid $UID --gid $GID wagtailio]: exit code: 4
make: *** [rebuild] Error 17
```

Reply:
Should be fixed by https://github.com/wagtail/wagtail.org/pull/265/commits/c9d247fd2f59bab0706cadb6c133612a54b77bba